src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$hand-transform-origin: bottom center;
+$hand-bottom: 50%;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted #000 1vmin;
+  border-radius: 50%;
+
+  &__hand-minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: $hand-bottom;
+    animation: rotate-minute 3600s infinite linear;
+    transform-origin: $hand-transform-origin;
+    animation-timing-function: steps(60);
+  }
+
+  &__hand-second {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    bottom: $hand-bottom;
+    background-color: #2c8000;
+    animation: rotate-second 60s linear infinite;
+    transform-origin: $hand-transform-origin;
```

Comment:
The 'transform-origin' property value should be consistent with the changes suggested for line 28. Use 'center bottom' instead of 'bottom center' for consistency and to follow conventional order.